playbooks/ansible/roles/samba.install/tasks/centos.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Install required rpms
+  when: config.install.samba.repo is defined
+  block:
+    - name: Install samba and ctdb
+      when: inventory_hostname in groups['cluster']
+      yum:
+        name:
+          - samba
+          - ctdb
+
+    - name: Install samba client/test
+      when: inventory_hostname in groups['clients']
+      yum:
+        name:
+          - samba-client
+          - samba-test
+
+- name: Install common dependencies
```

Comment:
I'll check but I doubt we have it actively maintained.

Reply:
For now I have added another bootstrap script for centos as we don't have version specific one for centos stream 9. Even otherwise the dependency list in bootstrap script from samba sources seems to be much longer than what is required at least with sit-environment and sit-test-cases.